tests/sles4sap/sap_deployment_automation_framework/prepare_ssh_config.pm

```diff
@@ -0,0 +1,65 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Summary:  Executes setup of HanaSR scenario using SDAF ansible playbooks according to:
+#           https://learn.microsoft.com/en-us/azure/sap/automation/tutorial#sap-application-installation
+
+use base 'sles4sap_publiccloud_basetest';
+use warnings;
+use strict;
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+use sles4sap::console_redirection;
+use sles4sap_publiccloud;
+use sles4sap::azure_cli qw(az_keyvault_list);
+use sles4sap::sap_deployment_automation_framework::inventory_tools;
+use sles4sap::sap_deployment_automation_framework::deployment qw(sdaf_ssh_key_from_keyvault);
+use sles4sap::sap_deployment_automation_framework::naming_conventions
+  qw(get_sdaf_inventory_path convert_region_to_short get_workload_vnet_code sut_private_key_path generate_resource_group_name);
+
+sub test_flags {
+    return {fatal => 1, publiccloud_multi_module => 1};
+}
+
+sub run {
+    my ($self, $run_args) = @_;
+    select_serial_terminal;
+
+    # Connect serial to Deployer VM to get inventory file
+    connect_target_to_serial();
+
+    my $jump_host_user = get_var('REDIRECT_DESTINATION_USER', 'azureadm');
+    my $jump_host_ip = get_required_var('REDIRECT_DESTINATION_IP');
```

Comment:
I like to keep as much of the `get_required_var` as possible at the beginning of the test module. So that if job has been triggered with some missing variables, it will fails as soon as possible before doing anything else

Reply:
Yeah I agree. Fixed.